bookwyrm/static/css/bookwyrm.css

```diff
@@ -30,6 +30,30 @@ body {
     min-width: 75% !important;
 }
 
+/** Utils
```

Comment:
Is there a more specific word than _Utils_ in this case? :)

Maybe a reason why classes are named this way would be helpful for others, for example.

Reply:
What do you think of `Utilities not covered by Bulma`?

Reply:
That would give more hint already about why classes are named this way, yep. :)